src/styles/blocks/page.scss

```diff
@@ -0,0 +1,12 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+}
+
+p {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
It's a good practice to move global styles, such as box-sizing, to a separate file (for example, _reset.scss or _normalize.scss). This way, your styles are more organized and easier to maintain.